examples/create-react-app/src/App.js

```diff
@@ -1,5 +1,7 @@
 import React, { useState } from 'react'
 import { GraphQLClient, ClientContext } from 'graphql-hooks'
+// eslint-disable-next-line no-unused-vars
```

Comment:
Was including this intended?

Reply:
yeah, the APQ issue with `crypto` not being polyfilled with webpack 5 only surfaces when the middleware is imported, once #847 is complete, that example could be used for the acceptance tests